content/blog/angular-state-management-with-ngxs/index.md

```diff
@@ -0,0 +1,233 @@
+---
+title: "Angular State Management With NGXS"
+date: "2020-09-07"
+coverImage: "title-image.png"
+author: "Akshaya Sharma"
+tags: ["Angular","NGXS","Redux"]
+description: "NGXS is a state management pattern + library for Angular. It acts as a single source of truth for your application's state, providing simple rules for predictable state mutations."
+---
+In this article, we will learn about state management in Angular with NGXS. Here we will understand the basic concept of NGXS and will create a simple TO-DO App.
```

Comment:
use of will is not required again.  Just `and create a simple TO-DO App` should work.